### PR TITLE
CGPROD 2067 Carousel tabs

### PR DESCRIPTION
**Pull request by FostUK**
Disables accessible elements for previous and next when they are hidden

### CODE REVIEW

File: test/components/how-to-play.test.js [hunk 1]
```diff
@@ -75,7 +74,13 @@ describe("How To Play Screen", () => {
         howToPlayScreen.scene = { key: "test-select" };
         howToPlayScreen.game = { canvas: { parentElement: "parent-element" } };
         howToPlayScreen.navigation = { next: jest.fn() };
+        howToPlayScreen.sys = {
+            queueDepthSort: jest.fn(),
+        };
         howToPlayScreen.setLayout = jest.fn(() => mockLayout);
+        Object.defineProperty(howToPlayScreen, "layout", {
+            get: jest.fn(() => mockLayout),
```
Comment (Leonie-) [inline test/components/how-to-play.test.js]: Not sure why this mock is needed? Will have a look...

Comment (Leonie-) [reply test/components/how-to-play.test.js]: Seems like it isn't?

Comment (Leonie-) [reply test/components/how-to-play.test.js]: Sorry - that should be line 77-79

File: test/mock/dom-element.js [hunk 2]
```diff
@@ -29,6 +29,7 @@ const domElement = () => {
         addEventListener: jest.fn(),
         removeEventListener: jest.fn(),
         childNodes: [],
+        update: jest.fn(),
```
Comment (Leonie-) [inline test/mock/dom-element.js]: This is not needed either...

File: src/components/how-to-play.js [hunk 3]
```diff
@@ -29,9 +29,9 @@ export class HowToPlay extends Screen {
             : this.add.image(0, -170, `${this.scene.key}.title`);
 
         if (this.theme.howToPlay) {
-            this.buttonLayout = this.setLayout(["overlayBack", "audio", "settings", "previous", "next"]);
+            this.setLayout(["overlayBack", "audio", "settings", "previous", "next"]);
         } else {
-            this.buttonLayout = this.setLayout(["home", "audio", "pause", "previous", "next", "continue"]);
+            this.setLayout(["home", "audio", "pause", "previous", "next", "continue"]);
```
Comment (Leonie-) [inline src/components/how-to-play.js]: Is this logic fork still used? Isn't it always How To Play now?

File: src/components/how-to-play.js [hunk 4]
```diff
@@ -47,14 +47,17 @@ export class HowToPlay extends Screen {
...
 
     setButtonVisibility() {
-        this.buttonLayout.buttons.previous.visible = Boolean(!this.theme.howToPlay || this.currentIndex !== 0);
+        this.layout.buttons.previous.visible = Boolean(!this.theme.howToPlay || this.currentIndex !== 0);
 
         const isNotLastPage = this.currentIndex + 1 !== this.choiceSprites.length;
-        this.buttonLayout.buttons.next.visible = Boolean(!this.theme.howToPlay || isNotLastPage);
+        this.layout.buttons.next.visible = Boolean(!this.theme.howToPlay || isNotLastPage);
+
+        this.layout.buttons.previous.accessibleElement.update();
```
Comment (Leonie-) [inline src/components/how-to-play.js]: Do we need tests for these new lines?

File: test/components/how-to-play.test.js (hunk 1 above)
Comment (FostUK) [reply test/components/how-to-play.test.js]: ah! Good spot. Was a fix for an intermediate issue I left behind. Updated :)

File: src/components/how-to-play.js (hunk 3 above)
Comment (FostUK) [reply src/components/how-to-play.js]: That's a good point. Looks like Alexa added it in originally - will double check with her when she gets in.

File: src/components/how-to-play.js (hunk 4 above)
Comment (FostUK) [reply src/components/how-to-play.js]: Visibility is tested in the fixed tests below.
Testing update is called feels like the wrong thing to check. Will have a think...

File: src/components/how-to-play.js (hunk 3 above)
Comment (FostUK) [reply src/components/how-to-play.js]: Yeah uneeded due to new select screen :)

File: src/components/how-to-play.js (hunk 4 above)
Comment (FostUK) [reply src/components/how-to-play.js]: Ended up testing update is called.
Not sure that's the best thing but it's mocked so prob all that can be done.

**Pull request merged**